unlock-app/src/components/content/event/EventVerification.tsx

```diff
@@ -21,38 +20,31 @@ export const EventVerification = ({
   const { data: event } = useEvent({ slug: slug })
 
   return (
-    <BrowserOnly>
-      <AppLayout authRequired={true} showHeader={false}>
-        {event && (
+    event && (
```

Comment:
Maybe use `if(event)` abive and return null if false?

Reply:
sounds neat! updating...